_pages/oidc.md

```diff
@@ -36,17 +36,18 @@ sidenav:
 
 ## Getting started
 
-### Choosing an authentication method
+### Client authentication
 
-Login.gov supports two ways of authenticating clients: **private_key_jwt** and **PKCE**.
+Login.gov supports only **private_key_jwt** for client authentication.
 
 - **private_key_jwt** (preferred for **web apps**)
   The client sends a [JSON Web Token][jwt], or JWT, signed with a private key (minimum length of 2048 bits) when requesting access tokens. The corresponding public key is registered with the IdP ahead of time, similar to SAML.
 
-- **PKCE** (preferred for **native mobile apps**)
-  Short for [Proof Key for Code Exchange by OAuth Public Clients](https://tools.ietf.org/html/rfc7636) and pronounced "pixy." In this method, the client sends a public identifier as well as a hashed random value generated by the client.
+**client_secret** is not supported by Login.gov because of security reasons. It requires managing a shared secret in two places: the client and the server, where **private_key_jwt** flow involves only sharing public keys with the server.
 
-Other implementations of OpenID Connect use the "implicit flow" or the **client_secret** param, but Login.gov does not support those methods. The implicit flow is [not recommended by the OAuth group][oauth-implicit-deprecated] for security reasons. Similarly **client_secret** is not supported by Login.gov because of security reasons. It requires managing a shared secret in two places: the client and the server, where **private_key_jwt** flow involves only sharing public keys with the server and **PKCE** only has a one-time secret.
+Clients that are unable to use **private_key_jwt** (e.g. native mobile apps) must use PKCE, short for [Proof Key for Code Exchange by OAuth Public Clients](https://tools.ietf.org/html/rfc7636) and pronounced "pixy." In this method, the client sends a public identifier as well as a hashed random value generated by the client.
```

Comment:
I think this worked better as an item in the bulleted list, since it's our alternative option to private key jwt. Also for this line, I think the emphasis should be on PKCE rather than jwt:

```suggestion
Clients that are unable to use private_key_jwt (e.g. native mobile apps) must use **PKCE**, short for [Proof Key for Code Exchange by OAuth Public Clients](https://tools.ietf.org/html/rfc7636) and pronounced "pixy." In this method, the client sends a public identifier as well as a hashed random value generated by the client.
```

Reply:
The change was based on the idea that PKCE is explicitly not a form of client authentication, and including it as an alternative to private_key_jwt makes it look like it is.